lib/tapioca/compilers/dsl/active_support_current_attributes.rb

```diff
@@ -0,0 +1,120 @@
+# typed: true
+# frozen_string_literal: true
+
+require "parlour"
+
+module Tapioca
+  module Compilers
+    module Dsl
+      # `Tapioca::Compilers::Dsl::ActiveSupportCurrentAttributes` decorates RBI files for all
+      # subclasses of `ActiveSupport::CurrentAttributes`
+      #
+      # To add attributes see https://api.rubyonrails.org/classes/ActiveSupport/CurrentAttributes.html
+      #
+      # For example, with the following singleton class
+      #
+      # ~~~rb
+      # class Current < ActiveSupport::CurrentAttributes
+      #   extend T::Sig
+      #
+      #   attribute :account
+      #
+      #   def helper
+      #     # ...
+      #   end
+      #
+      #   sig { params(user_id: Integer).void }
+      #   def authenticate(user_id)
+      #     # ...
+      #   end
+      # end
+      # ~~~rb
+      #
+      # this generator will produce an RBI file with the following content:
+      # ~~~rbi
+      # # typed: strong
```

Comment:
I added the documentation myself and left this as `strong`. Do we prefer to teach usage of `true`?

Reply:
Yes, let's continue using `typed: true` please.